Sources/InternxtSwiftCore/Services/HttpAPI/DriveAPI.swift

```diff
@@ -272,4 +272,20 @@ public struct DriveAPI {
 
     }
     
+    public func syncToken(currentAuthToken: String, deviceToken: String, type: String, debug: Bool = false) async throws -> SyncTokenResponse  {
```

Comment:
`syncToken` sounds too generic, and we are not "syncing" it, rather we are registering it, what we are registering is a `PushDeviceToken` here, so I would call this `registerPushDeviceToken`, adjust the types names correctly